web/cypress.config.ts

```diff
@@ -0,0 +1,11 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: process.env.CYPRESS_baseUrl || "http://localhost:5173",
```

Comment:
I'd be tempted to default this to :8000 as we'll soon have tests that hit the API

Reply:
https://github.com/ucl-arc-tre/portal/pull/33/commits/27ac4d700343daf874602455e0bfa00ade24a20d

The tests now fall back to `localhost:8000` . The readme instructions have also been simplified to reflect this.